user-org-util/src/main/resources/externalResource.properties

```diff
@@ -1 +1,2 @@
-# This file will be used to set to env variable so if env variable is not set to system PropertyCache will read properties from this file.
\ No newline at end of file
+# This file will be used to set to env variable so if env variable is not set to system PropertyCache will read properties from this file.
+sunbird_m2_repo_path=file://home/amit/.m2/repository
```

Comment:
I think you didn't want to checkin this.

Reply:
i added this for reference , that we need to put all our env key at one place for developer or devops reference